NeoMathEngine/test/src/learn/LUFactorizationTest.cpp

```diff
@@ -15,6 +15,7 @@ limitations under the License.
 
 #include <TestFixture.h>
 
+#if FINE_ARCHITECTURE( FINE_X86 ) || FINE_ARCHITECTURE( FINE_X64 ) //1 - CMathEngineQRFactorizationTest.Precalc.cpu (Failed) -- DARWIN macOS arm64
```

Comment:
Same here. It's better to mention the reason of failure (MKL).

Reply:
fixed